Formula/tarantool.rb

```diff
@@ -39,8 +37,13 @@ def install
     args << "-DENABLE_DIST=ON"
     args << "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}"
     args << "-DREADLINE_ROOT=#{Formula["readline"].opt_prefix}"
+    args << "-DENABLE_BUNDLED_LIBCURL=OFF"
     args << "-DCURL_INCLUDE_DIR=#{sdk}/usr/include"
     args << "-DCURL_LIBRARY=/usr/lib/libcurl.dylib"
```

Comment:
I don't think this is correct then. I'd expect it to be 
```suggestion
    args << "-DCURL_INCLUDE_DIR=#{Formula["curl"].opt_include}"
    args << "-DCURL_LIBRARY=#{Formula["curl"].opt_lib}/libcurl.dylib"
```

Reply:
Seems, that the new issue occurred on OSX 13 build with it:
```
/usr/bin/sandbox-exec -f /private/tmp/homebrew20201102-74692-yws255.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tarantool.rb --verbose
==> /usr/local/Cellar/tarantool/2.5.2.0/bin/tarantool /private/tmp/tarantool-test-20201102-74693-6rwmzr/test.lua
dyld: Library not loaded: /usr/local/opt/curl/lib/libcurl.4.dylib
  Referenced from: /usr/local/Cellar/tarantool/2.5.2.0/bin/tarantool
  Reason: Incompatible library version: tarantool requires version 12.0.0 or later, but libcurl.4.dylib provides version 7.0.0
Error: tarantool: failed
An exception occurred within a child process:
  BuildError: Failed executing: /usr/local/Cellar/tarantool/2.5.2.0/bin/tarantool /private/tmp/tarantool-test-20201102-74693-6rwmzr/test.lua
```

Reply:
Alexander, it is not only build-time dependency, but runtime as well (we'll need the library on the target system). Let's remove `:build`.

Reply:
Ok.